src/app/reskin/components/stepper/deposit/status.tsx

```diff
@@ -40,7 +40,7 @@ export function TxPendingStatus({ bitcoinTxId }: { bitcoinTxId: string }) {
       <div>
         ( Estimation{" "}
         <HyperLink href={`${PUBLIC_MEMPOOL_URL}/tx/${bitcoinTxId}`}>
-          5min
+          1 hour
```

Comment:
This is supposed to be 20 minutes for deposits.

Reply:
it used to take signers some time to confirm it, I remember matteo adjusting private mainnet to be 3 blocks so even then it's 30 minutes

Reply:
That's for withdrawals, here we are talking deposits

Reply:
okay will change

Reply:
fixed

Reply:
@djordon it might simply be that @hozzjss was referring to a chance to withdrawals -- not to deposits.

Reply:
I should've refreshed my page, I didn't need to write all of that.